articles/concepts/Glossary.md

```diff
@@ -12,31 +12,42 @@ uid: microsoft.quantum.glossary
-|Pauli Operators|The `X`, `Y` and `Z` quantum gates.|
-|Prelude|The set of primitive and classical operations and functions defined by each individual target machine, rather than at the Q# level.|
-|Quantum Circuit|The representation of a program for a quantum computer. See the <xref:microsoft.quantum.concepts.circuits> section for more details.|
-|Quantum State|A representation of the qubits in the system. This is usually denoted as a complex column vector. For more information, see <xref:microsoft.quantum.concepts.vectors>. |
-|Qubit|Unit of quantum storage. See the <xref:microsoft.quantum.concepts.qubit> section for more details.|
-|Repeat-until-success|A quantum algorithm that probabilistically succeeds. Upon failure, the routine will re-try until successful (or a limit has been reached). |
-|Target machine|A compilation target that lowers an abstract quantum program towards hardware or simulation. This typically include re-writes for many purposes including gate replacement, encoding for error correction, geometric layout and others.|
-|Tuple|Comma separated types grouped together via parenthesis. |
-|User-defined type|Collection of built-in or previously defined types that may be referred to as a single unit.|
+|<a id="adjoint"></a>Adjoint|The complex conjugate transpose of an [operation](xref:microsoft.quantum.glossary#operation). For operations that implement a [unitary](xref:microsoft.quantum.glossary#unitary) operator, the adjoint is the inverse of the operation and is indicated by a dagger symbol. For example, if the operation `U` represents the unitary operator $U$, then `Adjoint U` represents $U^\dagger$. For more information, see [Adjoint](xref:microsoft.quantum.language.file-structure#adjoint).|
+|<a id="ancilla"></a>Ancilla|A [qubit](xref:microsoft.quantum.glossary#qubit) that serves as temporary memory for a quantum computer and is allocated and de-allocated as needed.  For more information, see [Multiple qubits](xref:microsoft.quantum.concepts.multiple-qubits).|
+|<a id="bell-state"></a>Bell state|One of four specific maximally [entangled](xref:microsoft.quantum.glossary#entanglement) [quantum states](xref:microsoft.quantum.glossary#quantum-state) of two qubits. The four states are defined $\ket{\beta_{ij}} = (\mathbb{I} \otimes X^iZ^j) (\ket{00} + \ket{11}) / \sqrt{2}$. A Bell state is also known as an [EPR pair](xref:microsoft.quantum.glossary#epr).|
+|<a id="bloch"></a>Bloch sphere|A graphical representation of a single-[qubit](xref:microsoft.quantum.glossary#qubit) [quantum state](xref:microsoft.quantum.glossary#quantum-state) as a three-dimensional real-valued vector. For more information, see  [Visualizing Qubits and Transformations using the Bloch Sphere](xref:microsoft.quantum.concepts.qubit#visualizing-qubits-and-transformations-using-the-bloch-sphere).|
```

Comment:
I'm not so sure how we define it as real-valued. It can contain phase information of the qubit's coefficients.

Reply:
I assume this is just referring to the Bloch vector ([described here](https://en.wikipedia.org/wiki/Bloch_sphere)), which is a real-valued vector. But I agree, maybe here this could say "as a point in a three-dimensional unit sphere" instead of "as a three-dimensional real-valued vector"? The [cross-reference](https://docs.microsoft.com/en-us/quantum/concepts/the-qubit#visualizing-qubits-and-transformations-using-the-bloch-sphere) discusses the real-valued Bloch vector anyway.